src/actions/forecast.actions.js

```diff
@@ -0,0 +1,20 @@
+import { forecastConstants } from '../constants';
+import { forecastPeriodService } from '../services';
+
+
+export const forecastActions = {
+	fetchForecast,
+};
+
+function fetchForecast(){
+    return async (dispatch,currentState)=>{
+        const {lat, lng} = currentState().city.latLng;
+        try{
+            const forecast = await forecastPeriodService.getForecastByLatLng(lat, lng);
+            dispatch({ type: forecastConstants.GET_BY_LATLNG, forecast:forecast });
+        } catch (e){
+            console.log(e);
+            dispatch({ type: forecastConstants.GET_BY_LATLNG });
```

Comment:
As far as you basically don't do any action on `catch` but hide of an error I would remove `try/catch` for now

Reply:
+